src/app/systelab-components/combobox/abstract-combobox.component.ts

```diff
@@ -205,6 +209,15 @@ export abstract class AbstractComboBox<T> implements AgRendererComponent, OnInit
 	}
 
 	public ngOnInit() {
+		this.suppressKeyboardEvent = function(params) {
+			const KEY_TAB = 9;
+
+			const keysToSuppress = [KEY_TAB];
+			const event = params.event;
+			const key = event.which;
+			const suppress = keysToSuppress.indexOf(key) >= 0;
```

Comment:
This could be written in less rows of code which will be clearer

Reply:
Done. Good catch!!!